src/pages/getinvolved.jsx

```diff
@@ -13,71 +23,187 @@ import hackathon from '../../public/images/hackathon.png'
 import logo8 from '../../public/images/OI.png'
 import logo9 from '../../public/images/pacman.png'
 import roadmap from '../../public/images/roadmap.png'
-import orcas from '../../public/images/srkw2-10.jpg'
+import topbanner from '../../public/images/srkw2-25.jpg'
 import logo1 from '../../public/images/twt.png'
 import logo11 from '../../public/images/ws_logo.png'
 import getinvolvedStyles from '../styles/getinvolved.module.css'
 
-export const getinvolved = () => {
+const TopScreen = styled('div')(({ theme }) => ({
```

Comment:
Yes thanks, I can use Box. So should I use styled(Box) if more lines for styling are needed? Because using Box with lots of properties in  'sx' may not look good.

Reply:
@ShwetKhatri2001  I've been looking at more MUI code recently and just wanted to let you know I was wrong about this one (sort of). Turns out it doesn't really matter at all, makes no difference as far as I can tell. If you haven't imported `Box`, `div` might technically be a tiny bit better because you could have a smaller bundle size, but it's really insignificant.

I see you already changed it to `Box`, so no worries, just leave it because it's absolutely not important. Sorry for the confusion!